docs/cudf/source/Working-with-missing-data.ipynb

```diff
@@ -0,0 +1,3032 @@
+{
```

Comment:
<p>Following this section, it would also be inline and helpful on how we can replace NA/NAN with other values.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/7010/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>We have a section below for replacing <code>NA</code>/<code>NaN</code>   values: "Filling missing values: fillna". I have covered the content there.</p>